german/finger-ins-loch/hoelzernes-Paddel/hoelzernes-Paddel.md

```diff
@@ -1,2 +1,9 @@
 Das hoelzerne Paddel erweist sich als sehr marode. Damit schaffen Sie es niemals bis ans Ufer. Nichtsdestotrotz 
-sind Sie vorangekommen und sehen nun schon das Ufer. Sie hoeren Musik.
\ No newline at end of file
+sind Sie vorangekommen und sehen nun schon das Ufer. Sie hoeren Musik.
+
+Langsam naehern Sie sich dem Ufer an, als Sie eine grosse Anzahl spaerlich bekleideter Menschen sehen, die wie in Ekstase und unter Drogen zu lauter Musik tanzen. Was machen Sie?
+
+Sie
+… winken wild und rufen so laut Sie koennen, um auf sich [aufmerksam zu machen](aufmerksam_machen/aufmerksam_machen.md)
+
+...paddeln vorsichtig in eine nahegelegene Bucht, um sich das Ganze aus [sicherer Entfernung](sicherer_Entfernung/sicherer_Entfernung) anzuschauen. 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.